lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -12,7 +12,15 @@ class << self
 
         sig { returns(RunnerClient) }
         def create_client
-          new
+          if File.exist?("bin/rails")
+            new
+          else
+            warn(<<~MSG)
+              Ruby LSP Rails failed to locate bin/rails in the current directory: #{Dir.pwd}"
+            MSG
+            warn("Server dependent features will not be available")
```

Comment:
We're trying to move away from `warn` and use `$stderr.puts` instead, so that we're not affected by the `VERBOSE` configuration. Can we please switch?

Reply:
Should we make a dedicated PR to switch everything at once then? There are more than a dozen in the project atm.